app/actions/index.js

```diff
@@ -0,0 +1,114 @@
+import OSMSync from "../lib/osm-sync";
+import { timeout } from "../lib";
+
+const types = {
+  CLEAR_REMOTE_SURVEYS: "CLEAR_REMOTE_SURVEYS",
+  DISCOVERING_PEERS: "DISCOVERING_PEERS",
+  DISCOVERING_PEERS_FAILED: "DISCOVERING_PEERS_FAILED",
+  FETCHING_REMOTE_SURVEY: "FETCHING_REMOTE_SURVEY",
+  FETCHING_REMOTE_SURVEY_FAILED: "FETCHING_REMOTE_SURVEY_FAILED",
+  FETCHING_REMOTE_SURVEY_LIST: "FETCHING_REMOTE_SURVEY_LIST",
+  FETCHING_REMOTE_SURVEY_LIST_FAILED: "FETCHING_REMOTE_SURVEY_LIST_FAILED",
+  RECEIVED_REMOTE_SURVEY_LIST: "RECEIVED_REMOTE_SURVEY_LIST",
+  RECEIVED_REMOTE_SURVEY: "RECEIVED_REMOTE_SURVEY"
+};
+
+// fallback to 10.0.2.2 when connecting to the coordinator (host's localhost from the emulator)
+const COORDINATOR_FALLBACK_IP = "10.0.2.2";
+const COORDINATOR_FALLBACK_PORT = 3210;
+
+export default types;
+
+const getPeerInfo = (dispatch, callback) => {
+  dispatch({
+    type: types.DISCOVERING_PEERS
+  });
+
+  return new OSMSync().findPeers((err, peers) => {
```

Comment:
Yes, good call. Can we turn `findPeers` into a class method or not part of a class at all?